state/it/src/main/java/com/expediagroup/streamplatform/streamregistry/TestUtils.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (C) 2018-2023 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.streamplatform.streamregistry;
+
+import org.testcontainers.utility.DockerImageName;
+
+public class TestUtils {
+  public static DockerImageName kafkaImageName = DockerImageName.parse("confluentinc/cp-kafka:7.4.0");
```

Comment:
Minor, would this be more usual idiomatic Java?
```suggestion
  public static final DockerImageName KAFKA_IMAGE_NAME = DockerImageName.parse("confluentinc/cp-kafka:7.4.0");
```